buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -0,0 +1,95 @@
+#!groovy
+pipeline {
+    agent {label "${params.MACHINE_LABEL}"}
+    parameters {
+		choice (choices: 'x64_linux\nx64_mac\ns390x_linux\nppc64le_linux\naarch64_linux', description: 'PLATFORM?', name: 'PLATFORM')
+		choice (choices: 'SE90\nSE80', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
+		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8', description: 'What is JVM Version?', name: 'JVM_VERSION')
+		choice (choices: 'openjdk\nsanity\nextended\ntest_systemtest\nruntest', description: 'What is test level?', name: 'TARGET')
```

Comment:
I kind of think it would be useful to allow for more than these choices.  Perhaps we should consider a string parameter, so that we can run individual test cases.  If we keep it as a choice parameter, I'd like to see perf, external, jck added as choices.

Reply:
Yes, updated and also put examples in description.